source/client/gui/screens/IngameBlockSelectionScreen.cpp

```diff
@@ -69,6 +71,12 @@ bool IngameBlockSelectionScreen::isAllowed(int slot)
 
 void IngameBlockSelectionScreen::init()
 {
+	m_btnPause.m_width = 40;
+	m_btnPause.m_xPos = 0;
+	m_btnPause.m_yPos = 0;
+    if (m_pMinecraft->isTouchscreen())
+        m_buttons.push_back(&m_btnPause);
```

Comment:
Why does the inventory screen need a pause button?

Reply:
iOS does not feature a back button. In Version 0.1.3, the first release of Minecraft PE to iOS, this is how you exited the game.

However, I do suggest you put it behind an "if is iOS" check.

See https://www.youtube.com/watch?v=Yhje1jnQucg&t=3m43s

Reply:
From the old comment:
> iOS does not feature a back button. In Version 0.1.3, the first release of Minecraft PE to iOS, this is how you exited the game.

Reply:
GitHub needs to add auto-refresh on comments. First I delete the old comment not knowing you replied to it, then I copy over your comment to my new one, not knowing you also did so before me! I'm going to stop commenting before I accidentally double-copy, something else, again.

Anyway, that is crazy that Mojang actually decided "yes, people should exit the game through the inventory."

Reply:
> Anyway, that is crazy that Mojang actually decided "yes, people should exit the game through the inventory."

True. I'm not even sure when they added the top right pause button. v0.3.2 has it: https://www.youtube.com/watch?v=7w4lzLwsWhk

Reply:
Made this iOS-only.